rocketchat/README.md

```diff
@@ -426,3 +429,35 @@ References:
 IFF you manually enabled ingress.federation.serveWellKnown (which was a hidden setting) before, during upgrade, disable it once before enabling it again.
 
 Chart contained a bug that would cause `wellknown` deployment to fail to update (illegal live modification of `matchLabels`).
+
+### To 6.25.0
+
+**This is only applicable if you are using Prometheus monitoring with ServiceMonitor.**
+
+The chart has been updated to use PodMonitor instead of ServiceMonitor for Prometheus metrics collection. If you were using ServiceMonitor before, you'll need to update your values.yaml file. Here's how to migrate:
```

Comment:
```suggestion
The chart has been updated to use PodMonitor instead of ServiceMonitor for Prometheus metrics collection. If you were using ServiceMonitor before, it is recommended to update your values.yaml file and use podMonitor instead. Here's how to migrate:
```